pkg/util/httprequest/httprequest_test.go

```diff
@@ -25,20 +25,7 @@ func TestSchemeHost(t *testing.T) {
 				},
 			},
 			expectedScheme: "https",
-			expectedHost:   "example.com:443",
-		},
-		"X-Forwarded-Port overwrites X-Forwarded-Host port": {
```

Comment:
Do you know which proxy can behave like this? For me it sounds like a terribly broken configuration.